### ISSUE TITLE
Some problem about dataset

### ISSUE THREAD

**Issue by woshisj** [BerkeleyAutomation/gqcnn#27]
![image](https://user-images.githubusercontent.com/32431758/41884229-14b44f0a-7925-11e8-913d-bc9d5628c455.png)

**Comment by jeffmahler** [comment]
@woshisj Apologies for the confusion and thank you for your interest in the expanded dataset. We only documented a subset of the dataset that is necessary to replicate results in the paper.

Here are some notes on the name conventions:
- _suffix "_raw" for an image tensor:_ This means that the image is not transformed (rotated and translated) into the grasp reference frame
- _suffix "_table" for an image tensor:_ This means that the image includes the table supporting the object. Images without this suffix do not contain the table
- _suffix "_labels"_: This means that the tensor contains integer values that identify which object, stable pose, or camera position the image was generated with. This is useful for generating custom dataset splits.

1.binary_ims_raw_00000.npz
Pixels with value 255 are on the object and pixels with value 0 are on the background table.

2.depth_ims_raw_00000.npz
The depth images before transformation. The object is masked out from the background. Please see points above.

3.depth_ims_raw_table_00000.npz
The depth images before transformation including the background table. Please see points above.

4.force_closure_00000.npz
Friction and collisions should have been considered for these metrics. We used a friction coefficient of 0.5.

Please keep in mind that force_closure does not actually measure dynamic stability. It is a statement of the existence of a static equilibrium between the contact forces applied through the robot fingertips and arbitrary external forces and torques on the object.

5.hand_poses_00000.npz
Sorry about the confusion. This is an artifact of our raw storage format...

Channels 0 and 1 are the pixel locations of the grasp in the "raw" images.
Channel 2 is the depth.
So yes you want to use those.

Channels 4 and 5 are an artifact of the fact that we used to always render images centered on an object of interest. We actually translated the camera around so the true camera center pixel was not on the object. These values tell you how much the image was shifted to put the object in the center of the image.

6.object_labels_00000
These are integers identifying the unique object in the image. Please see points above.

7.pose_labels_00000.npz and image_labels_00000.npz
These are integers identifying the unique object stable pose and camera position, respectively, corresponding to the training datapoint. Please see points above.
